### DISCUSSION TITLE
new idea

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#753] [Ideas]
I suggest working on developing staxrip and making the video produce in several quality automatically

To clarify more
The dimensions of the video are as follows
1920*
1280*
848*
640*
If the program is programmed to automatically output from 1080 to 360 quality
It will be a wonderful development of the program

**Comment by Dendraspis** [comment]
I don't think so.  😁 
I also don't think anybody would use it. For testing purposes: yes, but not more. And for testing you can set/change the resolution pretty quick by hand.

**Reply by mr-qansuh** [reply]
I am the one who made a mistake in the expression, I mean that he adds it as an option
I see it as a new and unique addition of its kind and evolution
important to the program
Because it is useful in compressing several video files
Or a series, instead of making an episode by episode, it brings out all the qualities that we specified at once

**You understand me**

**Reply by Dendraspis** [reply]
Therefore we already have templates and *File Batch*.  🥴

**Reply by mr-qansuh** [reply]
I konw 
but It will take less time if the feature is added to the program

**Reply by Dendraspis** [reply]
Sure, but StaxRip cannot handle every possible idea.
And converting a source with different resolutions is *extremely* special.

**Reply by mr-qansuh** [reply]
Thank you for your interest
and Can you say the best version of StaxRip so far without defects

**Reply by Dendraspis** [reply]
Always the newest. 🙂 
And v2.7.0 will be released in a few days.  😉

**Comment by JJKylee** [comment]
I don't think there will be much need for this 'feature' even if it is implemented _at the very high cost of overhauling the program source code_ in reality. High cost, almost no benefit.

If you can't wait to see this in action, I recommend to study the _[Command Line Interface](https://github.com/staxrip/staxrip/wiki/Command-Line-Interface)_ of StaxRip and how to combine it with batch processing in CMD or PowerShell.

**Reply by mr-qansuh** [reply]
I don't know that its cost is high, but I wanted it for the speed of production
